locale/en/download/package-manager.md

```diff
@@ -294,3 +295,11 @@ Using **[Scoop](https://scoop.sh/)**:
 ```bash
 scoop install nodejs
 ```
+
+## z/OS
+
+IBMŔG SDK for Node.js - z/OSŔG is available in two installation formats,
```

Comment:
Alternately, you can use the HTML entity
```suggestion
IBM&reg; SDK for Node.js - z/OS&reg; is available in two installation formats,
```

Reply:
@nschonni - thanks. its working fine